addons/l10n_uk/data/template/account.tax-uk.csv

```diff
@@ -1,47 +1,77 @@
-"","","","","","","","","","base","refund","-6","",""
-"","","","","","","","","","tax","refund","","",""
-"PT0","","0%","purchase","0%","percent","0.0","tax_group_0","True","base","invoice","+7","",""
-"","","","","","","","","","tax","invoice","","",""
-"","","","","","","","","","base","refund","-7","",""
-"","","","","","","","","","tax","refund","","",""
-"PT2","","Exempt","purchase","Exempt","percent","0.0","tax_group_0","True","base","invoice","+7","",""
-"","","","","","","","","","tax","invoice","","",""
-"","","","","","","","","","base","refund","-7","",""
-"","","","","","","","","","tax","refund","","",""
-"PT8","","20%","purchase","20% EU (NIR)","percent","20.0","tax_group_0","False","base","invoice","+9||+7","",""
-"","","","","","","","","","tax","invoice","+2","2201",""
-"","","","","","","","","","tax","invoice","-4","2201","-100"
-"","","","","","","","","","base","refund","-9||-7","",""
-"","","","","","","","","","tax","refund","-2","2201",""
-"","","","","","","","","","tax","refund","+4","2201","-100"
-"PT5","","5%","purchase","5%","percent","5.0","tax_group_5","True","base","invoice","+7","",""
-"","","","","","","","","","tax","invoice","+4","2201",""
-"","","","","","","","","","base","refund","-7","",""
-"","","","","","","","","","tax","refund","-4","2201",""
-"ST5","","5%","sale","5%","percent","5.0","tax_group_0","True","base","invoice","+6","",""
-"","","","","","","","","","tax","invoice","+1","2200",""
-"","","","","","","","","","base","refund","-6","",""
-"","","","","","","","","","tax","refund","-1","2200",""
-"ST4","","0%","sale","0% EU (NIR)","percent","0.0","tax_group_0","False","base","invoice","+8||+6","",""
-"","","","","","","","","","tax","invoice","","",""
-"","","","","","","","","","base","refund","-8||-6","",""
-"","","","","","","","","","tax","refund","","",""
-"PT7","","0%","purchase","0% EU (NIR)","percent","0.0","tax_group_0","False","base","invoice","+9||+7","",""
-"","","","","","","","","","tax","invoice","","",""
-"","","","","","","","","","base","refund","-9||-7","",""
-"","","","","","","","","","tax","refund","","",""
-"ST11","","20%","sale","20%","percent","20.0","tax_group_20","True","base","invoice","+6","",""
-"","","","","","","","","","tax","invoice","+1","2200",""
-"","","","","","","","","","base","refund","-6","",""
-"","","","","","","","","","tax","refund","-1","2200",""
-"PT11","","20%","purchase","20%","percent","20.0","tax_group_20","True","base","invoice","+7","",""
-"","","","","","","","","","tax","invoice","+4","2201",""
-"","","","","","","","","","base","refund","-7","",""
-"","","","","","","","","","tax","refund","-4","2201",""
+"id","description","invoice_label","type_tax_use","tax_scope","name","amount_type","amount","tax_group_id","active","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","repartition_line_ids/factor_percent"

```

Comment:
Are you modifying existing taxes ? if yes it will need a migration script for the localisation but also bumping up the version of the module if i'm not mistaken :smile: 

For the upgrade script, it's just try_loading the template, you can have an example here https://github.com/odoo/odoo/pull/177113